searches/recursive_substring_search.py

```diff
@@ -0,0 +1,40 @@
+# Recursive Python3 program to find if a given pattern is 
+# present in a text 
+
+def exactMatch(text, pat, text_index, pat_index): 
```

Comment:
The function name should be snake_case as described in CONTRIBUTING.md.
```suggestion
def exact_match(text, pat, text_index, pat_index): 
```
Type hints  and doctests are also required :)

Reply:
done